README.md

```diff
@@ -5,27 +5,32 @@ The story of this project begins when, due to COVID-19, we have converted our re
 I've quickly evaluated a couple of existing solutions, but they were either too big and complicated and/or too expensive. As I assumed that other people would have the same challenge I had, I decided to spend my after-hours time making an open-source tailored system for the need. Yes - it is free as speech, not as beer.
 
 ## What WARP can do
+
 - It allows people to book / change / unbook desks (or even parking stalls) in the office.
 - It allows people to check who else will be in the office.
 - It works on mobile.
 - All is done in an easy, visual way.
 - Generate a report of past bookings and export it to Excel file
 
 ## More advanced features
+
 - Seats can be limited to certain people, so other people cannot book them (it is called assigned seats).
 - Seats can be disabled, so people don't see them at all.
 - Multiple zones (maps) can be created, for example, floors or parking.
 - Zones can be grouped. One person can have only one seat booked simultaneously in a zone group (so you can have one group for floors and another group for parking stalls).
 - Admin(s) can book / modify / unbook seat for any user.
 - Full admin interface to add/remove/edit maps, zones, groups, and users.
 - SAML2.0 support - via Apache [mod_auth_mellon](https://github.com/latchset/mod_auth_mellon) module.
+- LDAP and Active Directory - via LDAP3 library.
 - Translations - currently, English and Polish are supported.
 
 ## What I'm not even planning to do
+
 - Approvals - the main goal of the system was to make it autonomous and management-free. So I don't intend to implement approval flows.
-- Timezone support - the selected time is always in the same timezone as a zone. It works well and is simple. But in case someone would like to have a couple of zones in different timezones and keep the `one person one seat at a given time` rule across these timezones, this will fail.
+- Timezone support - the selected time is always in the same timezone as a zone. It works well and is simple. But in case someone would like to have a couple of zones in different timezones and keep the`one person one seat at a given time` rule across these timezones, this will fail.
```

Comment:
```suggestion
- Timezone support - the selected time is always in the same timezone as a zone. It works well and is simple. But in case someone would like to have a couple of zones in different timezones and keep the `one person one seat at a given time` rule across these timezones, this will fail.
```